azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurerm.go

```diff
@@ -72,7 +72,19 @@ func ResourceServiceEndpointAzureRM() *schema.Resource {
 		ValidateFunc: validation.StringInSlice([]string{"WorkloadIdentityFederation", "ManagedServiceIdentity", "ServicePrincipal"}, false),
 	}
 
-	r.SchemaVersion = 2
+	r.Schema["workload_identity_federation_issuer"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "The issuer of the workload identity federation service principal.",
+	}
+
+	r.Schema["workload_identity_federation_subject"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "The subject of the workload identity federation service principal.",
+	}
+
+	r.SchemaVersion = 3
```

Comment:
No need to upgrade the schema state. These two properties are `computed`,  does not introduce any breaking changes to current state files

Reply:
Done, thanks. I wasn't 100% sure so added it anyway.